widgets/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,5 @@
+<resources>
+    <string name="req_card_video_btn">VIDEO</string>
+    <string name="req_card_photo_btn">PHOTO</string>
+    <string name="test_text">Test Text</string>
```

Comment:
We can remove all of this from here? We don't need `Test Text`. Also, `VIDEO` and `PHOTO` are probably not labels.